daemon/rpc_account.go

```diff
@@ -30,14 +57,21 @@ func (r *RPC) AccountInfo(ctx context.Context, _ *pb.Empty) (*pb.AccountResponse
 		accountInfo.Type = internal.CodeSuccess
 	}
 
-	dedicatedIPExpired, err := r.ac.IsDedicatedIPExpired()
+	accountInfo.DedicatedIpStatus = internal.CodeSuccess
+	dipServices, err := r.ac.GetDedicatedIPServices()
 	if err != nil {
-		log.Println(internal.ErrorPrefix, "checking dedicated IP expiration: ", err)
+		log.Println(internal.ErrorPrefix, "getting dedicated ip services: %w", err)
 		return &pb.AccountResponse{Type: internal.CodeTokenRenewError}, nil
-	} else if dedicatedIPExpired {
+	}
+
+	if len(dipServices) == 0 {
 		accountInfo.DedicatedIpStatus = internal.CodeNoService
```

Comment:
If the user has the service, but doesn't have a server selected would display no-service. is this ok?